routes/confirm.js

```diff
@@ -46,26 +22,23 @@ router.post('/', function(req, res, next){
 		}
 	}
 
-    var crust = data.crusts[req.body[crust]];
+    var crust = data.crusts[req.body['crust']];
```

Comment:
we were missing the '

without it, data.crusts[req.body[crust]] was always returning 'undefined'